source/developers-guide/rest-api/plugin-api-extension/index.md

```diff
@@ -75,7 +75,7 @@ This is our plugin bootstrap which subscribes to two events. For the one thing i
 `Enlight_Controller_Dispatcher_ControllerPath_Api` event to register the API controller and for 
 the other thing it uses the `Enlight_Controller_Front_StartDispatch` event to register an additional namespace for 
 our plugin. We do that, because then the API manager class can load our resource with `\Shopware\Components\Api\Manager::getResource('Banner')`.
-Since `Shopware 5.2.17` the last part is not necessary anymore. You can simply use the services.xml file to register new resources to Shopware.
+Since `Shopware 5.2.17`, both parts are not necessary anymore. You can simply use the services.xml file to register new resources to Shopware. The API controller will be found via autoloading.
```

Comment:
are you sure the autoloading of controllers happens with `5.2.17`, too?

I summon you @shyim :smile:

Reply:
I think so. 5.2.7 is the smallest tag for this commit https://github.com/shopware/shopware/commit/e181c8cba776118dcf92bbda51aca1e7478b1959#diff-96c7e4e98b5a46f19925922a7fd4f4c0

Reply:
Looks good for me. But I would add example where the file has to been